docs/section23/Rule23-9.md

```diff
@@ -40,16 +41,27 @@
     - create a variable for the maximum volume flow rate: `maximum_supply_flowrate = 0`  
     - loop through each supply fan: `for supply_fan in fan_system.supply_fans:`  
         -- add the supply fan design_airflow to the maximum_supply_flowrate: `maximum_supply_flowrate += supply_fan.design_airflow`  
+
+    - get the minimum volume flow rate of the proposed terminal for the same zone.  We only need to look at one zone because System-11 is a single-zone system by definition.  This is to do the 3rd check - that the minimum airflow is greater than the rate required to comply with local standards.  It is assumed that the proposed model meets the minimum requirements of local standards.  
+    - use get_dict_of_zones_and_terminal_units_served_by_hvac_sys to get a list of zones and terminal units served by this HVAC system: `zones_and_terminals_dict = get_dict_of_zones_and_terminal_units_served_by_hvac_sys(B_RMD)[hvac_system_id]`
+    - get the id of the zone served by this HVAC system - there should only be one zone because System-11 is, by definition, a single zone system: `zone_id = zones_and_terminals_dict["ZONE_LIST"][0]`
+    - get the zone in the proposed model: `zone_p = get_component_by_id(P_RMD, Zone, zone_id)`
+        - create a variable to hold the total minimum volume flow rate of all terminals in the proposed zone.  This will be an array of values, one for each hour of the year.  We will sum the minimum volume flowrate for each terminal by the availability of the terminal to take into account cases where the proposed zone has multiple terminals that do not operate at the same time: `minimum_volume_list_p = []`
+        - look at each terminal in the proposed zone: `for terminal_p in zone_p.terminals:`
+          - get the HVAC system associated with the terminal: `hvac_p = terminal_p.served_by_heating_ventilating_air_conditioning_system`
+          - get the fan system: `fan_system_p = hvac_p.fan_system`
+          - get the operation schedule hourly values: `operation_schedule_hourly_values_p = fan_system_p.operating_schedule.hourly_values`
+          - get the terminal minimum volume flowrate: `min_volume_p = terminal_p.minimum_airflow`
```

Comment:
I am having trouble following this. I thought the plan, based on the text above, was to sum the minimum flow across different terminals factoring in the operational schedule for the parent HVAC fan schedule. I do not see where this is happening in the code. Can you please help me understand.